packages/js/customer-effort-score/src/customer-effort-score.tsx

```diff
@@ -14,8 +14,14 @@ import { CustomerFeedbackModal } from './customer-feedback-modal';
 const noop = () => {};
 
 type CustomerEffortScoreProps = {
-	recordScoreCallback: ( score: number, comments: string ) => void;
-	label: string;
+	recordScoreCallback: (
+		score: number,
+		secondScore: number,
+		comments: string
+	) => void;
+	title: string;
+	firstQuestion: string;
```

Comment:
Just a note that these are currently hardcoded in fairly brittle fashion, but that was the approach decided on [in the original issue](https://github.com/woocommerce/woocommerce/issues/35124#issuecomment-1290948377).